.github/workflows/lint.yml

```diff
@@ -23,4 +20,5 @@ jobs:
           git config --global user.name "Github Actions"
```

Comment:
just a heads up I updated the `npm run lint` command you might wanna update that aswell

Reply:
> just a heads up I updated the npm run lint command you might wanna update that aswell

if the npm script is changed, this will change as well since it just runs npm run lint

Reply:
yea if you can update it to what you had in your original PR that would be great, think this is it

`"lint": "eslint ./src/**/*.ts ./src/*.ts --fix"`

Reply:
done